src/renderer/components/cluster-settings/components/cluster-name-setting.tsx

```diff
@@ -43,7 +45,7 @@ export class ClusterNameSetting extends React.Component<Props> {
   componentDidMount() {
     disposeOnUnmount(this,
       autorun(() => {
-        this.name = this.props.cluster.preferences.clusterName;
+        this.name = this.props.cluster.preferences.clusterName || this.props.entity.metadata.name;
```

Comment:
Not sure if this is needed, probably ok to have it.

Reply:
I found locally that for some reason `clusterName` prop was `undefined` for my minikube cluster. That was caused empty input field and errors in console.